tests/test_acl.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/acl'
+
+# test client for acl creation and deletion
+class TestAcl < CiscoTestCase
+  def setup
+    # setup runs at the beginning of each test
+    super
+    @acl_name_v4 = 'test-foo-v4-1'
```

Comment:
The `setup` method should really only have device cleanup logic. The `@acl_name_v4` and `@acl_name_v6` variables should just be defined inside each test method without the `@` symbol which by convention ruby uses for class instance variables.